alerta-covid/app/controllers/symptoms_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SymptomsController < ApplicationController
+
+  # GET /symptoms or /symptoms.json
+  def index
+    @symptoms = Symptom.all
```

Comment:
LGTM, but I could add a pagination maybe.

Reply:
hey @ramasoftdev and @ArielMD what do you think about using serializers to expose data?

Reply:
@vyk2rr how can we apply that?